website/content/en/docs/building-operators/ansible/quickstart.md

```diff
@@ -32,8 +32,7 @@ This guide walks through an example of building a simple memcached-operator powe
 Make sure to define `IMG` when you call `make`:
 
   ```sh
-  export OPERATOR_IMG="quay.io/example/memcached-operator:v0.0.1"
-  make docker-build docker-push IMG=$OPERATOR_IMG
+  make docker-build docker-push IMG="example.com/memcached-operator:v0.0.1"
```

Comment:
```suggestion
  make docker-build docker-push IMG="<some-registry>/memcached-operator:v0.0.1"
```

Reply:
This value literally defaults to `example.com/memcached-operator` in this example, see https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/Makefile#L32.

Reply:
Hi @estroz, 

See that the person needs to push the image to there registry. Otherwise, it does not work. So, based on the previous questions/issues raised about that the  `example.com` does not make clear it. PS.: Also, why someone that is following the quick start would like the testdata example?

IMO the test data example require to be `example.com` because we cannot flag/set any specific registry. So, after the person read the tutorial and check `<some-registry>/memcached-operator:v0.0.1` and then see `example.com/memcached-operator:v0.0.1"` in the sample then, that information is very clear too.

Reply:
There's an [entire section](https://github.com/operator-framework/operator-sdk/pull/4665/files#diff-904511a83c884e3a04cbd1183f9b91e6f730dc5cdeacdd00ba654e66545b8146R133) about this in the tutorial. Perhaps a note at the top of each quickstart, but nothing more. There's only so much hand-holding we can do here.